src/tests/unit/tests/common/components/file-issue-details-button.test.tsx

```diff
@@ -90,4 +90,22 @@ describe('FileIssueDetailsButtonTest', () => {
         bugActionMessageCreatorMock.verifyAll();
         expect(wrapper.getElement()).toMatchSnapshot();
     });
+
+    test('getDerivedStateFromProps clears state when issueTrackerPath set', () => {
+        const props = { issueTrackerPath: ' ' };
```

Comment:
If we check for trimmed value, this test will fail!

Reply:
If we were to check for trimmed value, we'd want to keep this test case (and change the expectation), and add another test case for a value that's not empty after trimming.